holoviews/core/dimension.py

```diff
@@ -41,6 +41,41 @@ def param_aliases(d):
     return d
 
 
+def as_dimension(dimension):
```

Comment:
Could this not simply be called ``dimension``?

Reply:
Nevermind...I see ``dimension`` is frequently used as name even though ``dim`` is a lot shorter.

Reply:
I think I prefer ``as_dimension`` or ``asdimension``, akin to numpy ``asarray``, ``dimension`` is far too commonly used as a variable name so it would cause issues and be confusing.

Reply:
How about ``asdim``?

Reply:
Can't say I prefer it but if you insist I'll do it.

Reply:
It is a lot shorter so I do prefer it..unless you have any specific objections to it?

Reply:
Okay, pushed.